lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -20,6 +20,11 @@
   // the end of the text
   width: calc(min(#{$max-width}, 100%));
   max-width: max-content;
+  position: relative;
+
+  &::after {
```

Comment:
Is this block actually necessary? I thought `::after` didn't show up until `content` was set on it.

Reply:
> Is this block actually necessary? I thought `::after` didn't show up until `content` was set on it.

You're right. `::after` does not show up unless `content` is set. So that block is unnecessary.
RSC-600-focused-radio-checkbox